petastorm/spark/spark_dataset_converter.py

```diff
@@ -234,7 +259,21 @@ def make_spark_converter(
     else:
         compression_codec = "uncompressed"
 
+    if _is_delta_available():
+        # pylint: disable=global-statement
+        global _use_delta
+        # pylint: enable=global-statement
+        _use_delta = True
```

Comment:
We don't need global var `_use_delta`, when we need check whether delta is available, call `_is_delta_available()` directly (because the import `delta` will be fast exception the first time import)